### PR TITLE
Avoid use common path as output path

### PR DESCRIPTION
**Pull request by YingChen1996**
# Description

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by D-W- — APPROVED** [review]

File: cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-We use a datastore with datapath as input folder in this example. In first step, we will mount folder `azureml://datastores/workspaceblobstore/paths/azureml/` using `uri_file` type, then create `hello-world.txt` in it, in the second step, we will print the content in `hello-world.txt` using `uri_file` type.
\ No newline at end of file
+We use a datastore with datapath as input folder in this example. In first step, we will mount folder `azureml://datastores/workspaceblobstore/paths/azureml/` using `uri_folder` type, then create `hello-world.txt` in it, in the second step, we will print the content in `hello-world.txt` using `uri_file` type.
```
Comment (wangchao1230) [inline cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/README.md:2]: update the folder here

File: cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ inputs:
 outputs:
   pipeline_sample_output_data:
     # format for data store uri: azureml://datastores/<datastore_name>/paths/<path>/<on>/<datastore>
-    path: azureml://datastores/workspaceblobstore/paths/azureml  
+    path: azureml://datastores/workspaceblobstore/paths/azureml/set_path_on_datastore
```
Comment (wangchao1230) [inline cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml:11]: may use 4b_datastore_datapath_uri as the path here?

File: cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml [hunk 3]
```diff
@@ -20,7 +20,7 @@ jobs:
     inputs:
       sample_input_data: 
         type: uri_folder
-        path: azureml://datastores/workspaceblobstore/paths/azureml  
+        path: azureml://datastores/workspaceblobstore/paths/azureml/
```
Comment (wangchao1230) [inline cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml:23]: also change this path?

File: cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/README.md (hunk 1 above)
Comment (YingChen1996) [reply cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/README.md:2]: seems this folder is input path in sample_input_data which is azureml://datastores/workspaceblobstore/paths/azureml/

File: cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml (hunk 3 above)
Comment (YingChen1996) [reply cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml:23]: reverted

File: cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml (hunk 2 above)
Comment (YingChen1996) [reply cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml]: updated

File: cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/README.md (hunk 1 above)
Comment (YingChen1996) [reply cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/README.md:2]: I think this folder is input data sample_input_data of the first step, I will update ```then create `hello-world.txt` in it```

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**